Runtime/Scripts/RGBotConfigs/RGEntity.cs

```diff
@@ -6,18 +6,23 @@
 
 namespace RegressionGames.RGBotConfigs
 {
+    [HelpURL("https://docs.regression.gg/studios/unity/unity-sdk/overview")]
```

Comment:
Woah! I wanna use this more often

Reply:
Definitely... as discussed in our 1-1 .. we need to update our docs site to have a little less top down tutorial style and a little more raw information/direct link style to make good use of this for Unity components